samples/unit-tests/axis/crosshairs/demo.js

```diff
@@ -77,6 +86,63 @@ QUnit.test('Crosshair on multiple axes (#4927)', function (assert) {
         'path',
         'Secondary axis has cross'
     );
+
+    chart.update({
+        yAxis: [{
+            ...yAxisCrosshair,
+            opposite: false
+        },
+        {
+            ...yAxisCrosshair,
+            opposite: false
+        }, {
+            ...yAxisCrosshair,
+            opposite: true
+        }, {
+            ...yAxisCrosshair,
+            opposite: true
+        }]
+    }, true, true);
+
+
+    chart.series[0].update({
+        data: sampleData
+    });
+
+    chart.series[1].update({
+        data: sampleData
+    });
+
+    chart.addSeries({
+        yAxis: 2,
+        data: sampleData
+    });
+
+    chart.addSeries({
+        yAxis: 3,
+        data: sampleData
+    });
```

Comment:
Two things here:

1. Instead of multiple updates, prefer just one `chart.update()` with all options
2. Whenever you run multiple updates, prefer setting `redraw` argument to `false`. Just the last one should have `redraw=true` (or after the last one use `chart.redraw()`). It is faster this way (multiple redraws vs one)

Reply:
Yes, thanks. It makes sense to put everything in one `chart.update()` and I will remember about redrawing the chart only once in the future, thanks!